src/Events.ts

```diff
@@ -38,12 +39,12 @@ export function combineEvents<
     return Object.assign({}, ...args)
 }
 
-export function createEventBus<C extends EventDeclaration>(args: { events: C, channels?: Channel[] }): C {
+export function createEventBus<C extends EventDeclaration>(args: { events: C, channels?: Channel[] }): EventDeclaration {
     const transports = (args.channels || []).map(c => new Transport(c))
-    return Object.keys(args.events)
-        .reduce((conn: C, slotName) => {
-            const config = args.events[slotName].config
-            conn[slotName] = connectSlot(slotName, transports as Transport[], config)
-            return conn
-        }, {} as any)
+    return Object
+        .keys(args.events)
+        .reduce((acc: EventDeclaration, name) => {
```

Comment:
I'm wary of calling things `name` because of `window.name`. It's not reserved but is shadowed... Maybe keep `slotName`?

Reply:
Ok, why not.